batch.go

```diff
@@ -107,7 +107,7 @@ func (batch *Batch) Read(b []byte) (int, error) {
 	batch.mutex.Lock()
 	offset := batch.offset
 
-	_, _, err := batch.readMessage(
+	_, _, _, err := batch.readMessage(
```

Comment:
Is it OK to discard the attributes here? It seems like we should decompress the content that has been written to the output buffer passed to `Read([]byte)` isn't it?

Reply:
Yes but I think we will need to implement Message Set for the decompression to really make sense.